coordinator/bridge/src/test/java/io/stargate/bridge/codec/ValueCodecTest.java

```diff
@@ -415,22 +416,24 @@ public static Stream<Arguments> invalidListValues() {
   }
 
   public static Stream<Arguments> vectorValues() {
+    Column.ColumnType vectorType = VectorType.of("org.apache.cassandra.db.marshal.FloatType", 4);
```

Comment:
I _think_ this is what `Conversion` produces (via cassandraType.getClass().getName()). Test does not (and can not) validate it so String passed can be anything. But test cannot depend on backend-specific element type either.